plugins/flutter_aepcore/lib/src/aepmobile_init_options.dart

```diff
@@ -0,0 +1,37 @@
+/// Configuration options for initializing the Adobe Mobile SDK.
+class InitOptions {
+  /// The App ID for the Adobe SDK configuration.
+  String? appId;
+  /// The file path for the local configuration file
+  String? filePath;
+  /// Flag indicating whether automatic lifecycle tracking is enabled
+  bool? lifecycleAutomaticTracking;
```

Comment:
Can you rename this to `lifecycleAutomaticTrackingEnabled` to make it consistent with Android and iOS?